dll/win32/shell32/precomp.h

```diff
@@ -45,6 +45,10 @@
 #undef ShellExecute
 #include <undocshell.h>
 
+// #if (_WIN32_WINNT >= 0x0600)
+#define ShellMessageBoxW ShellMessageBoxWrapW
```

Comment:
This is done so as to avoid modifying and replacing all places in the shell32 code where ShellMessageBoxW is used, into ShellMessageBoxWrapW.

Reply:
Note that without this change, there is a linker error where the linker is unable to find `_ShellMessageBoxW`.